pkgs/development/tools/build-managers/jam/default.nix

```diff
@@ -1,7 +1,9 @@
-{ lib, stdenv, fetchurl, bison, buildPackages }:
+{ lib, stdenv, fetchurl, bison, buildPackages, pkgsBuildTarget }:
```

Comment:
Where is the patch on `all-packages.nix`?

Reply:
Hmm, I'm not sure I understand what you're asking for -- should I have to set `pkgsBuildTarget` to something explicitly in the `callPackage` for this package in `all-packages.nix`?

`pkgsBuildTarget` seems to work as-is for me, i.e., when I include `jam` in a `nativeBuildInputs` for another package and then cross-compile that package, it correctly represents the set of packages that run on the builder and produce binaries for the cross-compilation target. Does that not work for you? :< If it helps, I was building #190105 as e.g. `pkgsCross.s390x.p4`.

Reply:
Sorry, I was confused. I believed the `ftjam = callPackage ...` line was absent.